crates/burn-tensor/src/tests/quantization/ops/remainder.rs

```diff
@@ -28,6 +65,42 @@ mod tests {
 
     #[test]
     fn should_support_remainder_float() {
+        let device = Default::default();
+        let lhs = TestTensor::<1>::from_data(
+            // [1.0, 2.0, 3.0, 4.0, 5.0]
+            TensorData::quantized(
+                vec![-77i8, -26, 25, 76, 127],
+                [5],
+                QuantizationStrategy::PerTensorAffineInt8(AffineQuantization::init(0.019623, -128)),
+            ),
+            &device,
+        );
+        let rhs = TestTensor::<1>::from_data(
+            // [1.4233, 2.7313, 0.2641, 1.9651, 0.5897]
+            TensorData::quantized(
+                vec![5i8, 127, -103, 56, -73],
+                [5],
+                QuantizationStrategy::PerTensorAffineInt8(AffineQuantization::init(0.010700, -128)),
+            ),
+            &device,
+        );
+
+        let output = lhs.remainder(rhs);
+        let expected = TensorData::from([
+            1.0007842779159546,
+            2.001568555831909,
+            0.05992525815963745,
+            0.06563401222229004,
+            0.2960371971130371,
+        ]);
```

Comment:
The expected result should be `[1.0, 2.0, 0.0949, 0.0698, 0.2824]`, minus the de/quantization errors. I think it makes it more clear if you declare the result of the actual operation as if it was computed in floating point representation, and then the precision can be adjusted for the `assert_approx_eq`.

This comment probably applies for other expected results in following tests, but lmk what you think otherwise.